libs/cart/guides/state.md

```diff
@@ -242,5 +242,71 @@ export class AppModule {}
 
 The only argument to `forRoot` is the configuration object. For more information, see `DaffCartStateConfiguration`.
 
-<!-- TODO: add dependency injectable reducers guide once we implement it -->
+## Dependency Injectable Reducers
+
+`@daffodil/cart/state` provides mechanisms for consuming applications and libraries to modify state reduction behavior.
+
+### Purpose
+
+`@daffodil/cart/state` consumers may wish to modify the state for the `daffCart` feature in a way not explicitly provided by `@daffodil/cart/state`. A common example is adding payment info to the cart payment object in response to non-`@daffodil/cart/state` actions.
+
+#### Before vs. After
+
+Both before and after reducer slots are provided.
+
+Custom before reducers run before the native `@daffodil/cart/state` reducers. Before reducers can be used to contribute data to state with the understanding that the data may be overwritten by subsequent reducers.
```

Comment:
It should also be noted that if a before reducers acts upon uninitialized state, that they must initialize state themselves.

Reply:
I removed before reducers entirely